pkg/components/exporter/exporter.go

```diff
@@ -45,6 +45,9 @@ const (
 	overviewDashboardName = "power-monitoring-overview"
 	nsInfoDashboardName   = "power-monitoring-by-ns"
 	DashboardNs           = "openshift-config-managed"
+	REDFISH_ARGS          = "-redfish-cred-file-path=/etc/redfish/redfish.csv"
+	REDFISH_CSV           = "redfish.csv"
+	REDFISH_ANNOTATION    = "kepler.system.sustainable.computing.io/redfish-secret-ref"
```

Comment:
could you please change this to follow the convention? 
If these constants aren't used outside the package, lets make them private.